sdk/servicebus/azure-messaging-servicebus/README.md

```diff
@@ -125,6 +125,17 @@ on which actual message transmission takes place. The namespace often serves as
 * A **[subscription][subscription_concept]** receives messages from a topic. Each subscription is independent and
   receives a copy of the message sent to the topic.
 
+### Service Bus Clients
+The builder [`ServiceClientBuilder`][ServiceClientBuilder] is used to create all the servicebus clients.
```

Comment:
servicebus -> Service Bus

Can we dump this into a table?

Reply:
Table I did try but it is crossing 120 char line limit and I can not break in new line as table format is not maintained.